web/html/src/components/datetimepicker.tsx

```diff
@@ -131,10 +131,10 @@ class TimePicker extends React.Component<TimePickerProps> {
       roundingFunction: (seconds, options) => seconds,
     });
     this._input?.on("change", () => {
-      const timepickerValue = this._input?.timepicker("getTime");
+      this._input?.timepicker("getTime");
```

Comment:
It isn't clear whether this is the right action here. If we get the time and then don't use it I don't think we need this call either, as long as it has no side effects. Perhaps @ncounter is more familiar with it?  

Personally I would recommend making the whole block empty with the comment `// Do nothing` so that it's clear that it's intentionally an empty block. The value in keeping the empty block is that if someone needs to change this code next, they won't have to hunt down the docs for the plugin.

Reply:
Nice catch! It's fixed now, thanks.

Reply:
I agree with you guys. I can see the same pattern few lines after where it is also used, but in these two cases I don't see the need not only for that line, but also for the entire event action if we do not react in anyway.
Maybe for completeness we should ask @lucidd if there is a particular reason why he added back in the day almost 5 years ago in [this commit](https://github.com/uyuni-project/uyuni/commit/e2f937e9f245df82c406b43e7bbc94e21432bb28).